src/readinglistbar.cpp

```diff
@@ -15,6 +15,16 @@ ReadingListBar::ReadingListBar(QWidget *parent) :
     ui(new Ui::readinglistbar)
 {
     ui->setupUi(this);
+
+    //to remove gap between RL components
+    if (layout()) {
```

Comment:
Why is this check necessary? I don't see setLayout() calls later so this state won't change.